### PR TITLE
Added missing documentation for indices/keys in go.get/set

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

**Review by AGulev — DISMISSED** [review]
👍

File: engine/gameobject/src/gameobject/gameobject_script.cpp [hunk 1]
```diff
@@ -560,6 +563,30 @@ namespace dmGameObject
      * ```lua
      * local speed = go.get("#player", "speed")
      * ```
+     *
+     * @examples
+     * Get an value in a material property array
```
Comment (britzl) [inline engine/gameobject/src/gameobject/gameobject_script.cpp:568]: a value

File: engine/gameobject/src/gameobject/gameobject_script.cpp [hunk 2]
```diff
@@ -735,6 +765,31 @@ namespace dmGameObject
      * ```lua
      * go.set("#player", "speed", 100)
      * ```
+     *
+     * @examples
+     * Set an vector4 in a material property array
```
Comment (britzl) [inline engine/gameobject/src/gameobject/gameobject_script.cpp:770]: a vector4

**Review by britzl — DISMISSED** [review]
Looks good. Small typo "an" vs "a"

**Pull request merged**